### PR TITLE
(fix): Fix styling on vitals header

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Remove white space on vitals header

### Bug
Screenshot 2024-09-24 at 15 04 56

### Fix
Screenshot 2024-09-24 at 15 06 07

### CODE REVIEW

File: packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
   justify-content: space-between;
   align-items: center;
   min-height: layout.$spacing-09;
-  margin: layout.$spacing-03 0 0 layout.$spacing-05;
+  padding: layout.$spacing-03 0 0 layout.$spacing-05;
```
Comment (denniskigen) [inline packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss:14]:
```suggestion
  padding: 0 0 0 layout.$spacing-05;
```

Thanks, @donaldkibet. We should probably remove the top margin altogether.

File: packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss [hunk 2]
```diff
@@ -11,11 +11,17 @@
...
+  padding-left: layout.$spacing-05;
   outline: none;
   pointer-events: visibleFill;
 }
 
+.emptyStateVitalsHeader {
+  @extend .vitalsHeader;
+  background-color: $openmrs-background-grey;
+  border-bottom: 1px solid $ui-03;
+}
```
Comment (denniskigen) [inline packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss:23]: I've taken the liberty of pushing a commit that fixes the background color of the empty state:

### Before (in the UAT distro)

CleanShot 2024-09-24 at 9  47 24@2x

### After

CleanShot 2024-09-24 at 9  36 40@2x

Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss:23]: Do the changes look reasonable to you?

Comment (donaldkibet) [reply packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss:23]: This look great. Thanks!

**Review by denniskigen — APPROVED** [review]
Thanks!

**Pull request merged**